### ISSUE TITLE
Unable to open learning-deep/deeplearning1/DeepLearning4-LectureNotes.ipynb

### ISSUE THREAD

**Issue by jonny86** [timdavidlee/learning-deep#4]
Hello Tim,
thanks for sharing all that notebooks .
I try to open learning-deep/deeplearning1/DeepLearning4-LectureNotes.ipynb

When open the notebook on github I get 'Sorry, something went wrong. Reload?'
When opening the notebook locally with jupyter notebook I get some json errors.

Number 3 and 5 seems to work fine.
Can you check on this ?

Thanks in advance

Kind regards,
jonny

**Comment by timdavidlee** [comment]
yes, I noticed that recently as well. It could be that the jupyter notebook is too big to buffer in github? I will try and split up the content sometime this weekend

Thanks for the notification

**Comment by bb78657** [comment]
I also tried:

Opening in Jupyter lab X
Opening in pycharm X
Opening the raw file in github, pasting into file on local machine X
Deleting (some of the) images from the notebook X

I cant work out a way to get the problem line that throws the notJSONError by jupyter notebook

**Comment by vadimio** [comment]
Nope, it is not a Jupyter. The file is actually broken. I have submitted a pull request. Feel free to accept and close the issue
